### PR TITLE
Feature/hexxle 62

### PR DESCRIPTION
**Pull request by denadex**

### CODE REVIEW

File: unity/Assets/Scripts/UnityMonoBehaviour/UnityUI.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class UnityUI : MonoBehaviour
+{
+    public void exitGame()
```
Comment (RicardoMonteiroSimoes) [inline unity/Assets/Scripts/UnityMonoBehaviour/UnityUI.cs:8]: rename function as this is ambiguous to exiting the entire "game", as in not to the actual round

**Review by RicardoMonteiroSimoes — CHANGES_REQUESTED** [review]
The button is close to the middle of the screen, maybe change that?

File: unity/Assets/Scripts/UnityMonoBehaviour/UnityUI.cs (hunk 1 above)
Comment (denadex) [reply unity/Assets/Scripts/UnityMonoBehaviour/UnityUI.cs]: fixed in 10d35704b2bdaa410737e6ed888afb3ba8b3769a

**Review by DennisBannerman — COMMENTED** [review]
Since this is based on Adrianas branch, I'd wait to merge this until her branch is merged. If there may have been some changes on Adriana's side by then, rebase this branch.

**Review by RicardoMonteiroSimoes — APPROVED** [review]

**Pull request merged**